src/main/external-resources/UMS.conf

```diff
@@ -596,6 +596,15 @@ disable_transcode_for_extensions =
 # troubleshooting when you are not able to edit the renderer configuration.
 # Default: ""
 force_transcode_for_extensions =
+
+# Disable all transcoding
+# -----------------------
+# Do not transcode under any circumstances.
+# Warning: This will probably mean that your renderer will fail to play many
+# files or fail to show subtitles, but there are cases where this is still 
```

Comment:
So finally, we share the idea that it's nice to have but need to be used with care and be widely advertised. Different way same obective.

So that's fine for me, as i share that it need to be carefully used, it's why i though about commenting this function as "testing/debugging" to make users carefull.